src/main/java/com/std/tothebook/controller/ThymeleafController.java

```diff
@@ -28,7 +28,7 @@ public String text(Model model) {
         User user = userService.getUser(1);
         model.addAttribute("user", user);
 
-        log.info("log test");
+        log.info("{}, log test", user);
```

Comment:
```suggestion
        log.info("{}, log test", user.getUserSeq());
```
user만 남기면 주소 값이 남습니다.